spec/factories/lists.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :list do
+    title "MyString"
```

Comment:
Consider using realistic test data where it's easy to, such as the list title here. It can occasionally help debugging and identifying previously unknown issues.